astropy/cosmology/core.py

```diff
@@ -99,13 +99,13 @@ class FLRW(Cosmology):
         Neff. Usually this means you must provide three neutrino masses unless
         you are considering something like a sterile neutrino.
 
-    name : str
-        Optional name for this cosmological object.
-
-    Ob0 : float
+    Ob0 : float or None
```

Comment:
No -- it does **not** use the default.  That's why it isn't float, optional.  And this is, in fact, the behavior I think we want.  Many literature cosmologies (those that don't come from the CMB) don't have Omb0 values associated with them.  And I don't think we want to just quietly use a default, because that is more likely to cause, ah, unexpected outcomes.

That, however, could be documented better.  The idea is that when you pass None it is happy -- unless you ask for Omb(z), in which case it complains.